sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -532,6 +549,18 @@ async def _load_feature_flags(self, **kwargs):
 
                 if self._feature_flag_refresh_enabled:
                     feature_flag_sentinel_keys[(feature_flag.key, feature_flag.label)] = feature_flag.etag
+                if feature_flag.filters:
+                    for filter in feature_flag.filters:
+                        if filter.get("name") in PERCENTAGE_FILTER_NAMES:
+                            filters_used[PERCENTAGE_FILTER_KEY] = True
+                        elif filter.get("name") in TIME_WINDOW_FILTER_NAMES:
+                            filters_used[TIME_WINDOW_FILTER_KEY] = True
+                        elif filter.get("name") in TARGETING_FILTER_NAMES:
+                            filters_used[TARGETING_FILTER_KEY] = True
+                        else:
+                            filters_used[CUSTOM_FILTER_KEY] = True
```

Comment:
even "name" not in filter?

Reply:
Name is a required field. I wouldn't expect it to be missing. Either way it's a custom filter, just an invalid one.

Reply:
https://github.com/Azure/AppConfiguration/blob/d0ef7d5507b9cee15893eead5861e3b1a82d3efd/docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json#L77